### PR TITLE
cdc,pkg: accommodate systemd TimeoutStopSec and k8s terminationGracePeriodSeconds

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

Issue Number: ref #4757 

### What is changed and how it works?

Add liveness to TiCDC.

TiCDC sets liveness to Stopping upon receiving SIGTERM.
The exit perdure wait TiCDC drains all its tables and ownership (not implemented yet). 

#### Tests 

 - Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hicqu** [comment]
LGTM

**Review by 3AceShowHand — APPROVED** [review]

File: cdc/api/v1/api.go [hunk 1]
```diff
@@ -704,7 +704,7 @@ func (h *OpenAPI) ListCapture(c *gin.Context) {
 // @Produce json
 // @Success 200,202
 // @Failure 500,400 {object} model.HTTPError
-// @Router	/api/v1/captures/drain [post]
+// @Router	/api/v1/captures/drain [put]
```
[the remark this answers is not in the document]
Comment (overvenus) [reply cdc/api/v1/api.go:707]: It's a correction. It have been changed into PUT by https://github.com/pingcap/tiflow/pull/6044.

**Pull request merged**